codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/AddBuiltinPlugins.java

```diff
@@ -67,6 +67,11 @@
                         .withConventions("@aws-sdk/middleware-expect-continue", "^0.1.0-preview.5",
                                         "AddExpectContinue", HAS_MIDDLEWARE)
                         .servicePredicate((m, s) -> s.getId().getName().equals("AmazonS3"))
+                        .build(),
+                RuntimeClientPlugin.builder()
+                                .withConventions("@aws-sdk/middleware-sdk-glacier", "^0.1.0-preview.7",
+                                                "AddGlacierApiVersion", HAS_MIDDLEWARE)
+                                .servicePredicate((m, s) -> s.getId().getName().equals("Glacier"))
```

Comment:
Probably safer to use the service ID here.

Reply:
fixed in 65c11d13d78988b8c00fa594899ff29bd0046122